packages/is-shallow-equal/src/objects.js

```diff
@@ -7,36 +7,24 @@
  * @return {boolean} Whether the two objects are shallow equal.
  */
 export default function isShallowEqualObjects( a, b ) {
-	if ( a === b ) {
-		return true;
-	}
```

Comment:
Same for `isShallowEqualObjects`.

Reply:
If it's a public API there doesn't seem to be much benefit to removing this. I would expect the JS runtime to be extremely fast at such comparisons (unless someone can tell me otherwise).

Reply:
If it's a public API there doesn't seem to be much benefit to removing this. I would expect the JS runtime to be extremely fast at such comparisons (unless someone can tell me otherwise).